e2e/cypress/fixtures/hooks/message_buttons.json

```
@@ -0,0 +1,28 @@
+{
+    "attachments": [
+        {
+            "pretext": "This is the attachment pretext.",
+            "text": "This is the attachment text for Interactive Message Buttons.",
+            "actions": [
+                {
+                    "name": "Ephemeral Message",
```

Maybe add 
```suggestion
                    "name": "Ephemeral Message",
                    "type": "button",
```

To be explicit?

---

Or even better, have both a test with and without this being set and confirm the behaviour is the same.